scripts/validate_pulsar_release.sh

```diff
@@ -29,16 +29,28 @@ if [[ ! $LOCAL ]]; then
     # Download the release tarballs
     for file in apache-pulsar-$VERSION-bin.tar.gz apache-pulsar-$VERSION-bin.tar.gz.asc \
     apache-pulsar-$VERSION-bin.tar.gz.sha512 apache-pulsar-$VERSION-src.tar.gz apache-pulsar-$VERSION-src.tar.gz.asc \
-    apache-pulsar-$VERSION-src.tar.gz.sha512 connectors/pulsar-io-cassandra-$VERSION.nar; do
-        wget -c $BASE_URL/$file
+    apache-pulsar-$VERSION-src.tar.gz.sha512; do
+        wget -nv -c $BASE_URL/$file
     done
 
+    mkdir connectors
+    pushd connectors
```

Comment:
thanks for the PR. This seems to break the validation script and it fails on line `cp pulsar-io-cassandra-$VERSION.nar apache-pulsar-$VERSION/connectors`. Would you be able to test the changes and fix the issue?

Reply:
I got the problem resolved.